cumulusci/robotframework/locators_54.py

```diff
@@ -0,0 +1,17 @@
+import copy
+
+from cumulusci.robotframework import locators_53
+
+lex_locators = copy.deepcopy(locators_53.lex_locators)
```

Comment:
Where does the naming convention `locators_xx` comes from?

Reply:
We've used that since I got here three years ago. I'm guessing Jason established this convention.

Reply:
@boakley - Is there a significance to the numbers 52, 53, and 54?

Reply:
Yes, they correspond to the API version. At startup, we grab the API version and then try to find a locator file specific to that version. Every few releases I move the locators to the current release file and remove the old files. I haven't done that for this release yet, but I probably should just to keep things tidy.